staging/src/k8s.io/cri-client/pkg/logs/logs_test.go

```diff
@@ -363,12 +358,12 @@ func TestParseLog(t *testing.T) {
 		t.Logf("TestCase #%d: %+v", c, test)
 		parse, err := getParseFunc([]byte(test.line))
 		if test.err {
-			assert.Error(t, err)
+			require.Error(t, err)
```

Comment:
@mmorel-35 Can you show `testifylint` error/warning that this change fixes? `continue` on the next line suggests that the test is expected to continue with the next test case. Using `require.Error` would stop the test.

Reply:
This case is different, I believe. Using assert would allow to run other test cases, which could be useful. 
However, it probably makes sense to do this change to make testifylint happy. I'm ok with this.